assets/scss/components/_button.scss

```diff
@@ -268,8 +268,8 @@ Class                   | Description
 .button--bordered {
 	@include buttonColor(
 		$bgColor: $C_contentBG,
-		$hoverColor: #F6F7F8, //TODO: change to $C_collectionBG once `swarm-constants` is available
-		$activeColor: darken(#F6F7F8, 3%)
+		$hoverColor: $C_lightGray,
```

Comment:
Nothing in this file is part of this PR, but I figured I'd update SASS vars after upgrading to a version of `swarm-sasstools` that pulls from `swarm-constants`